simpleble/src/backends/macos/PeripheralBase.mm

```diff
@@ -36,6 +37,10 @@
     return [internal underlying];
 }
 
+std::string PeripheralBase::address_type() {
+    throw Exception::OperationNotSupported();
```

Comment:
Due to code compatibility across platforms, it's better if these kinds of functions return some default value instead of blowing up, as it puts a lot of burden on the user to not use this feature under certain OSes.

I'd return `unknown` instead.

Reply:
I just saw what you did on Windows, let's go with `unspecified` instead.